indica/models/plasma.py

```diff
@@ -837,7 +833,7 @@ def __call__(self):
 
 
 class PlasmaProfiles:
```

Comment:
hasn't this been superseded by the new class you've built?

connected to my comment on the pull request for this, I'd anyway save that new class in this file.